jetty-ee10/jetty-ee10-tests/jetty-ee10-test-siwe-webapp/src/main/webapp/login.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Isn't this the same file as the one in `src/test/resources` of the etherium module?

Reply:
Yes and it is not needed here, deleted now.